Patches/Caffe/WindowsDownloadPrebuiltDependencies.cmake

```diff
@@ -0,0 +1,182 @@
+set(DEPENDENCIES_URL_1800_27 "https://github.com/willyd/caffe-builder/releases/download/v1.1.0/libraries_v120_x64_py27_1.1.0.tar.bz2")
```

Comment:
Nope. That's all BVLC, except what I changed and added to get it to work with our stuff.